zerver/views/users.py

```diff
@@ -78,21 +78,28 @@ def reactivate_user_backend(request: HttpRequest, user_profile: UserProfile,
 @has_request_variables
 def update_user_backend(request: HttpRequest, user_profile: UserProfile, user_id: int,
                         full_name: Optional[str]=REQ(default="", validator=check_string),
-                        is_admin: Optional[bool]=REQ(default=None, validator=check_bool)) -> HttpResponse:
-    target = access_user_by_id(user_profile, user_id, allow_deactivated=True, allow_bots=True)
+                        is_admin: Optional[bool]=REQ(default=None, validator=check_bool),
+                        profile_data: List[Dict[str, Union[int, str, List[int]]]]=
+                        REQ(default=None,
+                            validator=check_list(check_dict([('id', check_int)])))) -> HttpResponse:
+    target_user = access_user_by_id(user_profile, user_id, allow_deactivated=True, allow_bots=True)
```

Comment:
Why don't you split out a separate commit for renaming `target` to `target_user`?